pkg/alerts/alertmanager/types.go

```diff
@@ -0,0 +1,95 @@
+package alertmanager
+
+import (
+	"fmt"
+	"net/http"
+)
+
+var (
+	silenceStates = map[string]bool{"expired": true, "active": true, "pending": true}
```

Comment:
Maybe a comment or another data structure would help here, as the intent is not obvious.